app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -231,8 +237,7 @@ const FundingAndFeeStep = (props: Props) => {
             />
           </CurrenciesWrapper>
         )}
-        <WarningMessage />
-        <GridTransactionDetailsStyled noMarginTop={tokensAmount === 1}>
+        <GridTransactionDetailsStyled noMarginTop={false}>
```

Comment:
```suggestion
        <GridTransactionDetailsStyled noMarginTop={false}>
```

You could make this prop optional if you find it troublesome. Also you could remove it if it isn't being used anymore.